<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1082</width>
    <height>602</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1082</width>
    <height>602</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1082</width>
    <height>602</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Home</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>210</width>
      <height>210</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/furn.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>200</y>
      <width>210</width>
      <height>210</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/tech.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>200</y>
      <width>210</width>
      <height>210</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/office.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>200</y>
      <width>210</width>
      <height>210</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/all.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="furniture_button">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>420</y>
      <width>151</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
            border-radius: 10px;
            padding: 5px;
            background-color: #009688;
            color: white;
        }

        QPushButton:hover {
            background-color: #008080;
        }

        QPushButton:pressed {
            background-color: #006666;
        }</string>
    </property>
    <property name="text">
     <string>Furniture</string>
    </property>
   </widget>
   <widget class="QPushButton" name="technology_button">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>420</y>
      <width>151</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
            border-radius: 10px;
            padding: 5px;
            background-color: #009688;
            color: white;
        }

        QPushButton:hover {
            background-color: #008080;
        }

        QPushButton:pressed {
            background-color: #006666;
        }</string>
    </property>
    <property name="text">
     <string>Technology</string>
    </property>
   </widget>
   <widget class="QPushButton" name="office_button">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>420</y>
      <width>151</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
            border-radius: 10px;
            padding: 5px;
            background-color: #009688;
            color: white;
        }

        QPushButton:hover {
            background-color: #008080;
        }

        QPushButton:pressed {
            background-color: #006666;
        }</string>
    </property>
    <property name="text">
     <string>Office Supplies</string>
    </property>
   </widget>
   <widget class="QPushButton" name="everything_button">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>420</y>
      <width>151</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
            border-radius: 10px;
            padding: 5px;
            background-color: #009688;
            color: white;
        }

        QPushButton:hover {
            background-color: #008080;
        }

        QPushButton:pressed {
            background-color: #006666;
        }</string>
    </property>
    <property name="text">
     <string>View All</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>50</y>
      <width>1011</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <pointsize>28</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Welcome to Data Pirates Superstore</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>510</y>
      <width>1021</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color:grey
}</string>
       </property>
       <property name="text">
        <string>Â© Team Data Pirates </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../project.qrc"/>
 </resources>
 <connections/>
</ui>
